zazu/repo/commands.py

```diff
@@ -83,9 +87,63 @@ def clone(repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
-def init():
-    """Initialize repo directory structure."""
-    raise NotImplementedError
+@click.option('--nohooks', is_flag=True, help='does not install git hooks in the repo')
+@click.pass_context
+def init(ctx, nohooks):
+    """Initialize repo directory structure"""
+    def _zazu_yaml(repo_name, issue_tracker={}, stylers=[]):
+        """builds zazu.yaml file"""
+        if issue_tracker:
+            zazu_yaml_obj = issue_tracker
+        else:
+            zazu_yaml_obj = {}
+        if stylers:
+            zazu_yaml_obj['style'] = {key: {'options': ' '} for key in stylers}
+            click.echo('Reminder: please specify styler options in  zazu.yaml')
+        yaml.dump(zazu_yaml_obj, file('zazu.yaml', 'w'), default_flow_style=False)
+
+    def _zazu_config(repo_name):
+        """Configures zazu"""
+        if click.confirm("Configure zazu.yaml?", abort=True):
+            click.echo("Configuring Zazu")
+            if os.path.isfile('zazu.yaml'):
+                click.confirm('zazu.yaml file found, continuing will overwrite, continue?', abort=True)
+            trackers = zazu.util.get_plugin_list(zazu.issue_tracker.IssueTracker)
+            stylers = zazu.util.get_plugin_list(zazu.styler.Styler)
+            tracker_choice = zazu.util.pick(trackers.keys(), 'Pick an Issue Tracker', True)
+            tracker_dict = {}
+            if tracker_choice is not None:
+                owner = click.prompt('Please enter an owner for issues created from this repo')
+                tracker_dict['issueTracker'] = {'owner': owner, 'repo': repo_name, 'type': tracker_choice}
```

Comment:
This will not work for JIRA. We need to interrogate each issue tracker to see what fields to ask for here.